src/api/rest/v1/providers/controller.ts

```diff
@@ -0,0 +1,102 @@
+import { Request, Response } from 'express'
+import Providers from "../../../../providers"
+import { ProviderHandler } from '../../../../interfaces'
+import CONFIG from '../../../../config'
+
+const log4js = require("log4js")
+const logger = log4js.getLogger()
+
+const SCHEMA_SYNC_LOG = CONFIG.verida.schemas.SYNC_LOG
+
+/**
+ */
+export default class Controller {
+
+    public static async provider(req: Request, res: Response) {
+        const providerId = req.params.providerId
+        const providers = Object.keys(CONFIG.providers)
+
+        const results: any = []
```

Comment:
Strongly recommended to use a proper type

Reply:
Fixed